babel/src/test/resources/sql/postgresql.iq

```diff
@@ -63,6 +63,11 @@ EXPR$0
 2022-06-03 12:15:48.678
 !ok
 
```

Comment:
I expect there should be some documentation about the arguments of the to_char function.

Reply:
There is a function description of to_char in reference.md
```bash
| m o p | TO_CHAR(timestamp, format)                 | Converts *timestamp* to a string using the format *format*
```
Is this enough?

Reply:
![Screenshot_39](https://github.com/apache/calcite/assets/77189278/00f2e939-40f6-4ce9-b178-2d0b0b4d2d34)

Reply:
From this documentation the users have no way of knowing what the format strings are. It would be nice to either have a list of all of them, or a link to a document someplace else which describes them. Otherwise how are users supposed to use this function? In many cases the Calcite documentation is too sparse, and that cannot help the users, especially when we are dealing with functions which are not "standard".

Reply:
Postgres has extensive documentation, you can link to that: https://www.postgresql.org/docs/16/functions-formatting.html#FUNCTIONS-FORMATTING-DATETIME-TABLE

Reply:
Thank you very much for your patient reply. I have a question. Which part of the document should I add, reference.md?

Reply:
Yes. That file has grown very large, and it could use some refactoring, but at this point it's *the* place for such documentation.